### PR TITLE
Filter by tag

### PR DESCRIPTION
**Pull request by charliew-rsi**
Added front-end functionality for displaying a list of filter tags.

### CODE REVIEW

File: src/components/Home/MainView.js [hunk 1]
```diff
@@ -72,6 +18,9 @@ const mapDispatchToProps = dispatch => ({
 const MainView = props => {
   return (
     <div className="col-md-9">
+      <div>
```
[the remark this answers is not in the document]
Comment (charliew-rsi) [reply src/components/Home/MainView.js]: amendment made

File: src/components/Home/MainView.js [hunk 2]
```diff
@@ -85,7 +34,7 @@ const MainView = props => {
           <TagFilterTab tag={props.tag} />
         </ul>
       </div>
-
+      <div />
```
[the remark this answers is not in the document]
Comment (charliew-rsi) [reply src/components/Home/MainView.js]: amendment made

File: src/components/Home/filters.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        <button
+          className="btn btn-sm btn-primary"
+          type="button"
+          onClick={this.toggleFilters}
+        >
+          Toggle Filters
+        </button>
+
+        {filtersActive && (
+          <ul className="list-unstyled toggleUL">
```
[the remark this answers is not in the document]
Comment (charliew-rsi) [reply src/components/Home/filters.jsx]: converted to div

File: src/components/Home/filters.jsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  toggleFilters() {
+    this.setState(prevState => ({
+      filtersActive: !prevState.filtersActive
+    }));
+  }
+
+  render() {
+    const { filtersActive, tags } = this.state;
+    return (
+      <div>
```
[the remark this answers is not in the document]
Comment (charliew-rsi) [reply src/components/Home/filters.jsx]: Converted to React.Fragment

File: src/components/Home/filters.jsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import agent from "../../agent";
+import { fetchAllTags } from "../../services/article";
+
+export const YourFeedTab = props => {
```
[the remark this answers is not in the document]
Comment (charliew-rsi) [reply src/components/Home/filters.jsx]: Aforementioned components are now deprecated, will remove components from project.

File: src/components/Home/filters.jsx [hunk 6]
```diff
@@ -100,18 +45,18 @@ export class Filters extends React.Component {
     return (
       <div>
```
Comment (wesdollar) [inline src/components/Home/filters.jsx]: This div still needs to be replaced with `<React.Fragment>`

**Pull request merged**